examples/myrobot/myrobot.py

```diff
@@ -71,8 +64,5 @@ def run(self):
         except BusShutdownException:
```

Comment:
This try/except should be part of the framework, meaning it should be inside `Node`. Like `StopIteration` is part of for cycle.

Reply:
Can `MyRobot` be implemented by overriding `update()`? So that we have both examples of what can be overridden on `Node`.

Reply:
OK, fixed - removed `run()` used `update()` instead. Nicer, thanks.